packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/table/customcells/CheckboxCellComponent.java

```diff
@@ -40,7 +40,9 @@ public Element render() {
         String targetObjectKind = cellDescription.getTargetObjectKindProvider().apply(variableManager);
 
         Boolean value = cellDescription.getCellValueProvider().apply(variableManager, this.props.columnTargetObject());
-        CheckboxCellElementProps cellElementProps = new CheckboxCellElementProps(this.props.cellId(), this.props.checkboxCellDescription().getId(), targetObjectId, targetObjectKind, this.props.columnId(), value);
+        String tooltipValue = cellDescription.getCellTooltipValueProvider().apply(variableManager, this.props.columnTargetObject());
```

Comment:
FWIW, it looks like this will imply evaluating possibly hundreds/thousands of expressions on *every render* of a table, even though in practice 99.9% of the cells' tooltips will never be displayed.
Given that tooltips are only displayed on the fronted after a small delay anyway, would it be possible to only compute them lazily/on-demand though a dedicated query/data fetcher?

Reply:
I have opened [an issue](https://github.com/eclipse-sirius/sirius-web/issues/4925) for that but merge this in the meantime